app/presenters/vendor_api/note_presenter.rb

```diff
@@ -0,0 +1,24 @@
+module VendorAPI
+  class NotePresenter < Base
+    attr_reader :note
+
+    def initialize(version, note)
+      super(version)
+      @note = note
+    end
+
+    def to_json(_)
```

Comment:
```suggestion
    def as_json
```

Reply:
@cwrw `as_json` was the original method name, @despo suggested the `schema` / `to_json` convention as an easier way to extend. I'm ambivalent.